extensions/script/CMakeLists.txt

```diff
@@ -37,18 +37,20 @@ if (NOT DISABLE_PYTHON_SCRIPTING)
     	find_package(PythonLibs 3.0 REQUIRED)
     endif()
 
-    include_directories(${PYTHON_INCLUDE_DIR})
-    include_directories(SYSTEM ../../thirdparty/pybind11/include)
-
-    include_directories(python)
     add_definitions(-DPYTHON_SUPPORT)
 
     file(GLOB PY_SOURCES  "python/*.cpp")
     add_library(minifi-python-extensions STATIC ${PY_SOURCES})
-
+    target_include_directories(minifi-python-extensions SYSTEM PRIVATE ${PYTHON_INCLUDE_DIR})
+    target_include_directories(minifi-python-extensions SYSTEM PRIVATE ../../thirdparty/pybind11/include)
+    target_include_directories(minifi-python-extensions PRIVATE python)
     target_link_libraries(minifi-python-extensions ${LIBMINIFI})
     target_link_libraries(minifi-python-extensions ${PYTHON_LIBRARIES})
+    target_compile_options(minifi-python-extensions PUBLIC $<$<CXX_COMPILER_ID:Clang>:-fsized-deallocation>)
```

Comment:
just out of curiosity, why is this needed?

Reply:
Due to the error mentioned in this PR: https://github.com/pybind/pybind11/pull/1957
Since pybind11 is now updated and they included a workaround in their codebase, I will remove it.